### PR TITLE
Jon additions to case studies workflow in contribution guidelines

### PR DESCRIPTION
**Pull request by jonleung**

### CODE REVIEW

**Comment by jonleung** [comment]
@zachlatta, can I have you eyes on this plz? Thanks!

**Comment by jonleung** [comment]
Not ready for merge lol

**Comment by jonleung** [comment]
@paked, I'd love your feedback on this as well!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,9 +73,50 @@ going through:
...
 This section will cover any contribution guidelines specific to
-[Labs](meta/labs).
-
-### Case Studies
+[Labs](meta/labs) and any others that choose to participate.
+
+### Contributing case studies
+
+First make sure that you've read the
+[`README.md` in the `case study` directory](/case_studies/README.md)
```
Comment (zachlatta) [inline CONTRIBUTING.md]: Should there be a period at the end of this line?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -73,9 +73,50 @@ going through:
...
+
+#### Before club events
+
+1. Follow steps 1 - 5 in the
+   ["case study contribution walkthrough" section below](#case-study-contribution-walkthrough)
+2. Find your club's folder (located inside of [`case_studies`](/case_studies))
+3. Inside your club's folder, make sure that the event you are planning has it's
+   own folder (create one if it does not already exist)
```
Comment (zachlatta) [inline CONTRIBUTING.md]: `it's` -> `its`

Comment (jonleung) [inline CONTRIBUTING.md]: Cool. I made a fix that you'll see in the next version.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jonleung) [inline CONTRIBUTING.md]: Cool. I made a fix that you'll see in the next version.

**Comment by jonleung** [comment]
@zachlatta, lemme know when I should squash my commits!

**Pull request merged**